controlplane/kubeadm/internal/controllers/remediation_test.go

```diff
@@ -1877,10 +1877,20 @@ func withHealthyEtcdMember() machineOption {
 
 func withUnhealthyEtcdMember() machineOption {
 	return func(machine *clusterv1.Machine) {
+		newConditions := machine.Status.Conditions.DeepCopy()
+		machine.Status.Conditions = newConditions
```

Comment:
what is this for?

Reply:
This is out-of-date code and removed in the latest commit.